### ISSUE TITLE
[Bug]: A multi-artist album is being split into multiple albums

### ISSUE THREAD

**Issue by 1nfin1te** [navidrome/navidrome#3480]
### I confirm that:

- [X] I have searched the existing [open AND closed issues](https://github.com/navidrome/navidrome/issues?q=is%3Aissue) to see if an issue already exists for the bug I've encountered
- [X] I'm using the latest version (your issue may have been fixed already)

### Version

0.53.3 

### Current Behavior

In my disk, each album has its own folder, but within the same album, there may be works by multiple artists, for example: one hit song per artist. This type of album, after being scanned, ends up being split into different albums based on the artist. I have read articles [#2688](https://github.com/navidrome/navidrome/discussions/2668) and [#2728](https://github.com/navidrome/navidrome/issues/2728); the release dates within the albums are all the same. I have tried setting ND_SCANNER_GROUPALBUMRELEASES=true and also ND_SUBSONICARTISTPARTICIPATIONS=true, but neither of these solutions has resolved my issue。
![image](https://github.com/user-attachments/assets/3f00feee-036d-492d-a6f9-738d5ccdab5e)

### Expected Behavior

Albums can be formed based on tracks having the same album name in the media metadata. To avoid dirty data, it could also ensure that only tracks with the same album name and release date within the same folder are grouped into an album

### Steps To Reproduce

1.Add multi-artist album to media folder
2.Scan
3.Multiple albums appear

### Environment

```markdown
- OS:Docker
- Browser:Chrome
- Client:Web interface
```

### How Navidrome is installed?

Docker

### Code of Conduct

- [X] I agree to follow Navidrome's Code of Conduct

**Comment by 1nfin1te** [comment]
[2688](https://github.com/navidrome/navidrome/discussions/2668)

**Comment by 1nfin1te** [comment]
> Please see this FAQ article: https://www.navidrome.org/docs/faq/#-i-have-an-album-with-tracks-by-different-artists-why-is-it-broken-up-into-lots-of-separate-albums-each-with-their-own-artist
> 
> TLDR: You should do either:
> 
> * set the `albumartist` for all files in the same album
> * set the `compilation` to `1` for all files in the same album

I found that setting the `albumartist` indeed solved the issue. Thank you very much.
